src/Components/Publishing/Display/DisplayAd.tsx

```diff
@@ -31,20 +33,28 @@ export const DisplayAd: SFC<DisplayAdProps> = props => {
   }
 
   const [width, height] = adDimension.split("x").map(a => parseInt(a))
+  const ad = (
+    <GPT
+      adUnitPath={`/21805539690/${props.adUnit}`}
+      targeting={props.targetingData}
+      slotSize={[width, height]}
+      onSlotRenderEnded={e => {
+        isAdEmpty = e.isEmpty
+      }}
+    />
+  )
```

Comment:
could probably return 
```
if (isAdEmpty) {
  return null
}
```
further up, before all of this stuff initializes. Then can avoid potential confusion in return statement.